python3/cloudwatch/src/lambda_function.py

```diff
@@ -161,6 +178,12 @@ def lambda_handler(event, context):
     additional_data = _get_additional_logs_data(aws_logs_data, context)
     shipper = LogzioShipper()
 
+    whitelist_regexps.extend(_get_list_from_env_var("WHITELIST", "WHITELIST_SEPARATOR"))
+    blacklist_regexps.extend(_get_list_from_env_var("BLACKLIST", "BLACKLIST_SEPARATOR"))
+
+    if whitelist_regexps and blacklist_regexps:
+        logger.warning("Both whitelist and blacklist are defined. Whitelist will be used and blacklist will be ignored")
```

Comment:
Please have this check before the processing of the logs, and if both are used, log an error and exit